print-dhcp6.c

```diff
@@ -784,6 +790,43 @@ dhcp6opt_print(netdissect_options *ndo,
 			ND_PRINT(")");
 			break;
 
+		case DH6OPT_BOOTFILE_URL:
+			if (optlen < 2) {
+				ND_PRINT(" ?)");
```

Comment:
Please make this message something that allows the user understand what is wrong.

Reply:
There are 20 other instances of `ND_PRINT(" ?)");`, so it seemed like it was a known pattern in this file.  If you do want the new code to be inconsistent with the old, what about using `ND_PRINT(" [|opt])";`, to use the common truncation marker and try to evoke that it's the option that's truncated?

Reply:
That's a fair point. Alright, let's keep this error reporting consistent for now.